sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -4,12 +4,20 @@
 
 ### Features Added
 
-### Breaking Changes
+- A new boolean keyword argument `uamqp_transport` has been added to sync and async `ServiceBusClient` constructors which indicates whether to use the `uamqp` library or the default pure Python AMQP library as the underlying transport.
```

Comment:
```suggestion
- A new optional boolean keyword argument `uamqp_transport` has been added to sync and async `ServiceBusClient` constructors which indicates whether to use the `uamqp` library or the default pure Python AMQP library as the underlying transport.
```

Reply:
Not a huge thing, but I wonder if optional (like typing) might imply that it can be None, rather than only True or (default) False?

Reply:
fair, but even if they did pass in `None` it would get resolved to the default value :) ... we can leave it as is.